forem/forem
# Implement Replit Unified Embed

## What type of PR is this? (check all applicable)


## Description
This PR implements the [unified embed tag](https://github.com/forem/forem/issues/15099) for Replit embeds.

## Related Tickets & Documents
closes https://github.com/forem/forem/issues/15570

## QA Instructions, Screenshots, Recordings

Example Replit URLs:
https://replit.com/@msarit/Shell-Challenge#index.html
https://replit.com/@msarit/Shell-Challenge

Replit Inputs:
@msarit/Shell-Challenge#index.html
@msarit/Shell-Challenge

- As a user, create a Replit liquid tag using this format: `{% embed <replit-url> %}`. Please use the example URLs provided above. The Replit Liquid Tag should render properly.
- Also create Replit liquid tag using the old method: `{% replit <replit-input> %}`. This should also work, as I am leaving the old implementation in place for now.

### UI accessibility concerns?
None

## Added/updated tests?


## [Forem core team only] How will this change be communicated?
Considering the project as a whole, I shall update the Liquid Tag documentation at the end. Since I am not changing the current implementation, the present documentation is still valid. Plus, I will avoid updating the docs piecemeal, and risk confusing users.

## [optional] Are there any post deployment tasks we need to perform?
None


## How this PR makes me feel:
![Ratatouille cooking](https://media.giphy.com/media/wNDa1OZtvl6Fi/giphy.gif)
